setup.py

```diff
@@ -95,13 +95,14 @@ def run(self):
 
 # Remove the env marker if/when pyarrow is available for Python3.9
 hdfs = ["pyarrow>=0.17.0;  python_version < '3.9'"]
+webhdfs = ["hdfs==2.5.8"]
```

Comment:
The library looks pretty good, but there are a few concerns:

1) Conda package for 2.5.8 hasn't been merged yet https://github.com/conda-forge/python-hdfs-feedstock :slightly_frowning_face: 

2) No wheels https://pypi.org/project/hdfs/#files :slightly_frowning_face:

Reply:
I agree with the concerns, unfortunately I couldn't find a library in PyPI that looked any better. The other ones I saw are in Beta or Alpha versions, and even less active than `hdfs`. A few examples: https://pypi.org/project/WebHDFS/, https://pypi.org/project/PyHDFS/, https://github.com/pywebhdfs/pywebhdfs. I'm open to switching libraries if a better one is found though (either in this PR or a different one).

I do have a use case at work where having having `webhdfs` support would allow us to get rid of `pyarrow` dependencies, and simplify our onboarding process. Not a pressing concern though, and regardless I'm also happy to help with the implementation.

Reply:
Sounds good! :+1: Thank you!